elements/lisk-p2p/src/utils/miscellaneous.ts

```diff
@@ -14,26 +14,21 @@
  */
 import { hash } from '@liskhq/lisk-cryptography';
 import { isIPv4 } from 'net';
-import { P2PPeerInfo } from './p2p_types';
+import { P2PDiscoveredPeerInfo, P2PPeerInfo } from '../p2p_types';
 
 const SECRET_BUFFER_LENGTH = 4;
 const NETWORK_BUFFER_LENGTH = 1;
-const PREFIX_BUFFER_LENGTH = 1;
 const BYTES_4 = 4;
+const PREFIX_BUFFER_LENGTH = 1;
```

Comment:
why move this one?

Reply:
Move from where? It is related to `getIPBytes()`.

Reply:
from before BYTES_4 to after it. `LENGTH` ones group together and `BYTES` ones group together.